docs/wip/traffic/permissive_traffic_policy_mode.md

```diff
@@ -0,0 +1,343 @@
+    ```
+
+1. Deploy the `httpbin` service into the `httpbin` namespace after enrolling its namespace to the mesh. The `httpbin` service runs on port `14001`.
+
+    ```bash
+    # Create the httpbin namespace
+    kubectl create namespace httpbin
+
+    # Deploy httpbin service in the httpbin namespace
+    kubectl apply -f docs/example/manifests/samples/httpbin/httpbin.yaml -n httpbin
+    ```
+
+    Confirm the `httpbin` service and pods are up and running.
+
+    ```console
+    $ kubectl get svc -n httpbin
+    NAME      TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
+    httpbin   ClusterIP   10.96.198.23   <none>        14001/TCP   20s
+    ```
+
+    ```console
+    $ kubectl get pods -n httpbin
+    NAME                     READY   STATUS    RESTARTS   AGE
+    httpbin-5b8b94b9-lt2vs   2/2     Running   0          20s
+    ```
+
+1. Deploy the `curl` client into the `curl` namespace after enrolling its namespace to the mesh.
+
+    ```bash
+    # Create the curl namespace
+    kubectl create namespace curl
+
+    # Deploy curl client in the curl namespace
+    kubectl apply -f docs/example/manifests/samples/curl/curl.yaml -n curl
+    ```
+
+    Confirm the `curl` client pod is up and running.
+
+    ```console
+    $ kubectl get pods -n curl
+    NAME                    READY   STATUS    RESTARTS   AGE
+    curl-54ccc6954c-9rlvp   2/2     Running   0          20s
+    ```
+
+1. Confirm the `curl` client is able to access the `httpbin` service on port `14001`.
+
+    ```console
+    $ kubectl exec -n curl -ti "$(kubectl get pod -n curl -l app=curl -o jsonpath='{.items[0].metadata.name}')" -c curl -- curl -I http://httpbin.httpbin:14001
+    HTTP/1.1 200 OK
+    server: envoy
+    date: Mon, 15 Mar 2021 22:45:23 GMT
+    content-type: text/html; charset=utf-8
+    content-length: 9593
+    access-control-allow-origin: *
+    access-control-allow-credentials: true
+    x-envoy-upstream-service-time: 2
+    ```
+
+    A `200 OK` response indicates the HTTP request from the `curl` client to the `httpbin` service was successful.
+
+1. Confirm the HTTP requests fail when permissive traffic policy mode is disabled.
+
+    ```bash
+    # Assumes OSM is installed in the osm-system namespace
+    kubectl patch ConfigMap osm-config -n osm-system -p '{"data":{"permissive_traffic_policy_mode":"false"}}' --type=merge
+    ```
+
+    ```console
+    $ kubectl exec -n curl -ti "$(kubectl get pod -n curl -l app=curl -o jsonpath='{.items[0].metadata.name}')" -c curl -- curl -I http://httpbin.httpbin:14001
+    curl: (7) Failed to connect to httpbin.httpbin port 14001: Connection refused
+    command terminated with exit code 7
+    ```
+
+## Envoy configurations
+In permissive mode, OSM controller programs wildcard routes for client applications to communicate with services. Following are the envoy inbound and outbound filter and route configuration snippets from the `curl` and `httpbin` sidecar proxies.
```

Comment:
A brief description of exactly what's different between these envoy configs and what I would see if I had permissive mode disabled would be helpful. A line-by-line comparison would be too much, but something simple like "this object wouldn't exist" or pointing out a particular field to check in a sentence or two. I think that would help readers of this doc map these examples to their own workloads and help troubleshoot.

Reply:
I feel this should suffice for now. When we end up adding the SMI doc and config snippet, we can provide a link to what the config shouldn't look like.